### PR TITLE
YONE 3 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/spawning_pod.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Spawning Pod
+ManaCost:2 G
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 1 T Sac<1/Creature> | Origin$ Library | Destination$ Battlefield | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | RememberChanged$ True | ChangeType$ Creature.cmcEQX | ChangeNum$ 1 | SorcerySpeed$ True | AILogic$ SacAndUpgrade | SubAbility$ DBAnimate | SpellDescription$  Seek a creature card with mana value equal to 1 plus the sacrificed creature's mana value and put that card onto the battlefield. That creature is a Phyrexian in addition to its other types. Activate only as a sorcery.
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Types$ Phyrexian | Duration$ Permanent | SubAbility$ DBCleanup
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/spawning_pod.txt:5]: it probably needs to be AnimateSubAbility so it would enter the battlefield as Phyrexian

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/spawning_pod.txt]: RememberChanged is not needed

File: forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Phyrexian Harvester
+ManaCost:R R R R
+Types:Creature Phyrexian Horror
+PT:5/5
+K:Menace
+T:Mode$ DamageDone | ValidTarget$ Card.Self | Execute$ TrigSeek | TriggerDescription$ Whenever CARDNAME is dealt damage, seek that many nonland cards. At the beginning of your next end step, discard those cards.
+SVar:TrigSeek:DB$ ChangeZone | Origin$ Library | Destination$ Hand | AtRandom$ True | NoShuffle$ True | Mandatory$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ X | ChangeType$ Card.nonLand | RememberChanged$ True | SubAbility$ DBDelay
+SVar:DBDelay:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the next end step, discard those cards.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt:8]: The cleanup should happen as part of the normal trigger already

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt:8]: You still need to do it manually though

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt:8]: You mean between line 8 and 9 ?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt:8]: Yea that's when you don't need it anymore

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt]: nooo, don't overcomplicate it like that haha

just do it as normal sub after you created the delayed trigger

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt]: oops forgot you get set cleanup on subability

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/phyrexian_harvester.txt]: Remove `SubAbility$ DBDiscardExiled`

**Review by tool4ever — APPROVED** [review]

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**